Classes/Domain/Model/Changes/AbstractCopy.php

```diff
@@ -36,6 +36,7 @@ protected function finish(NodeInterface $node): void
         $updateNodeInfo->setNode($node);
         $updateNodeInfo->recursive();
         $this->feedbackCollection->add($updateNodeInfo);
+        $this->addNodeCreatedFeedback($this->getSubject());
```

Comment:
why do we need this line here? Rest looks understandable:)!

Reply:
This will select the new document as active, basically does the same as Redirect (from a UI behavior standpoint).
Fun fact: This is already included in Neos 9, but there the AbstractCopy does not exist anymore: https://github.com/neos/neos-ui/blob/9.0/Classes/Domain/Model/Changes/CopyBefore.php#L88

Reply:
```suggestion
        $this->addNodeCreatedFeedback($node);
```

okay in my test i had to change this line to use the `$node` as `getSubject` seems to be the old node which was copied which resulted in moving back to the copy source location instead of the target.